pkgs/development/tools/xcpretty/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
+  inherit ruby;
+  pname = "xcpretty";
+  gemdir = ./.;
+
+  passthru = {
+    updateScript = ./update;
+  };
+
+  meta = with lib; {
+    description     = "Flexible and fast xcodebuild formatter ";
+    homepage        = https://github.com/supermarin/xcpretty;
+    license         = licenses.mit;
+    maintainers     = with maintainers; [
```

Comment:
This style is a little unusual, but ultimately your call I think.

Reply:
Ah, you made me realize there was a trailing space in the description 😅

Reply:
Heh, I didn't even notice that.